grpc_unary_attestation/src/server.rs

```diff
@@ -59,19 +131,75 @@ where
 impl<F, S, L> UnarySession for AttestationServer<F, L>
 where
     F: 'static + Send + Sync + Clone + FnOnce(Vec<u8>) -> S,
-    S: std::future::Future<Output = anyhow::Result<Vec<u8>>> + Send + Sync + 'static,
+    S: std::future::Future<Output = anyhow::Result<Vec<u8>>> + Send + Sync,
     L: Send + Sync + Clone + LogError + 'static,
 {
     async fn message(
         &self,
         request: tonic::Request<UnaryRequest>,
-    ) -> Result<tonic::Response<UnaryResponse>, tonic::Status> {
+    ) -> anyhow::Result<tonic::Response<UnaryResponse>, tonic::Status> {
```

Comment:
This particular result is just a plain old Result, there is already structure in tonic::Status, so I don't think anyhow::Result helps here, does it?

Reply:
Happy to remove it. :) Just used it because my impression was were using anyhow everywhere